src/App/screens/Instructor/components/PaginatedLessonList/components/LessonSummary/index.js

```diff
@@ -0,0 +1,43 @@
+import React from 'react'
+import {truncate} from 'lodash'
+import Heading from '../../../../../../components/Heading'
+import Button from '../../../../components/Button'
+
+export default ({lesson}) => {
+
+  const nextStepForCurrentStates = {
+    accepted: {
+      label: 'Claim',
+      action: () => console.log('Next step action tapped'),
+    },
+  }
```

Comment:
Should let us easily add next steps to other lesson states in the future. If a next step exists, a button shows:
![image](https://cloud.githubusercontent.com/assets/5497885/20545040/1c195ad0-b0ca-11e6-9ff2-6421db06be2f.png)
![image](https://cloud.githubusercontent.com/assets/5497885/20545045/208f5326-b0ca-11e6-9eb2-94f6e04b43e5.png)